docs/guides/tutorials/getting_started/connect_to_data.rst

```diff
@@ -14,88 +14,71 @@ Let's configure your first Datasource, by following the next steps in the CLI in
     What data would you like Great Expectations to connect to?
         1. Files on a filesystem (for processing with Pandas or Spark)
         2. Relational database (SQL)
-    : 1
+    : 2
     
-    What are you processing your files with?
-        1. Pandas
-        2. PySpark
-    : 1
-    
-    Enter the path (relative or absolute) of the root directory where the data files are stored.
-    : my_data
+    Which database backend are you using?
+        1. MySQL
+        2. Postgres
+        3. Redshift
+        4. Snowflake
+        5. BigQuery
+        6. other - Do you have a working SQLAlchemy connection string?
+    : 2
     
     Give your new Datasource a short name.
-     [my_data__dir]: 
-    
-    Great Expectations will now add a new Datasource 'my_data__dir' to your deployment,
-    by adding this entry to your great_expectations.yml:
-    
-      my_data__dir:
-        data_asset_type:
-          class_name: PandasDataset
-          module_name: great_expectations.dataset
-        batch_kwargs_generators:
-          subdir_reader:
-            class_name: SubdirReaderBatchKwargsGenerator
-            base_directory: ../my_data
-        class_name: PandasDatasource
-    
+     [my_postgres_db]:
     
-    Would you like to proceed? [Y/n]: 
-
-That's it! You just configured your first Datasource!
-
-Before continuing, let's stop and unpack what just happened, and why.
-
-Why Datasources?
-----------------
+    Next, we will configure database credentials and store them in the `my_postgres_db` section of this config file: great_expectations/uncommitted/config_variables.yml:
```

Comment:
Maybe add a manual linebreak here? In my build-docs view, this scrolls off screen instead of wrapping.